website/docs/docs/collaborate/environments/dbt-cloud-environments.md

```diff
@@ -0,0 +1,211 @@
+---
+title: "dbt Cloud environments"
+id: "dbt-cloud-environments"
+---
+
+
+An environment in dbt Cloud determines how dbt Cloud will execute your project in both the dbt Cloud IDE and scheduled jobs. Critically, in order to execute dbt, environments define three variables:
+
+1. The version of dbt Core that will be used to run your project
+2. The warehouse connection information (including the target database/schema settings)
+3. The version of your code to execute
+
+For users familiar with development on the CLI, each environment is roughly analogous to an entry in your `profiles.yml` file, with some additional information about your repository to ensure the proper version of code is executed. More info on dbt core environments [here](/docs/collaborate/environments/dbt-core-environments.md).
+
+## Types of environments
+
+In dbt Cloud, there are two types of environments: deployment and development. Deployment environments determine the settings that will be used when jobs created within that environment are executed. Development environments determine the settings that will be used in the dbt Cloud IDE for that particular dbt Cloud Project. Each dbt Cloud project can only have one single development environment, but can have any number of deployment environments.
+
+|  | Development Environments | Deployment Environments |
+| --- | --- | --- |
+| Determines settings for | dbt Cloud IDE | dbt Cloud Job runs |
+| How many can I have in my project? | 1 | Any number |
+
+## Common Environment Settings
+
+To create a new dbt Cloud environment, navigate to the `Environments` page under the `Deploy` header menu, and click `Create Environment` . There is some commonality in settings between deployment and development environments, and some settings that are specific to each type.
```

Comment:
since this paragraph is a how-to instruction, suggest moving it to the "Create ...." sections. this can help with discoverability because people are used to procedures having a title that starts with a verb (_doing something_)

Reply:
since this is a common action regardless of the env type, would another header for `Create an environment` be appropriate?